sdk/appconfiguration/azure-data-appconfiguration/src/samples/java/com/azure/data/appconfiguration/AadAuthentication.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.data.appconfiguration;
+
+import com.azure.core.credential.TokenCredential;
+import com.azure.data.appconfiguration.models.ConfigurationSetting;
+
+/**
+ * Sample demonstrates how to use AAD token to build a configuration client.
+ */
+public class AadAuthentication {
+    /**
+     * Sample for how to use AAD token Authentication.
+     *
+     * @param args Unused. Arguments to the program.
+     */
+    public static void main(String[] args) {
+        // The endpoint can be obtained by going to your App Configuration instance in the Azure portal
+        // and navigating to "Overview" page. Looking for the "Endpoint" keyword.
+        String endpoint = "{endpoint_value}";
+
+        // Token Credential could be an AAD token which you can get from Identity
+        // or other service authentication service.
+        TokenCredential tokenCredential = null;
```

Comment:
I'd have an example link here on a credential you can obtain, like DefaultAzureCredential